.changeset/ten-crews-complain.md

```diff
@@ -0,0 +1,8 @@
+---
+'@openfn/language-salesforce': major
```

Comment:
I'm gonna call this a patch release.

it's technically a breaking fix.

But we have a major version coming soon which I expect jobs to migrate to. This patch just ensures that the 4.x branch is stable and behaves as originally intended.